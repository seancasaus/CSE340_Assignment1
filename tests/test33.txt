t983ud ((((a).(x))|(_))*).((b).(y)) , m983j ((_).((a)*)).(((x)|(_)).(x)) , qr432 (((_).((_)*))*).(x) #
" xbyaxbyaaaxaxaxby"