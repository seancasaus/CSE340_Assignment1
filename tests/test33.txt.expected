m983j , "x"
t983ud , "by"
t983ud , "axby"
m983j , "aaax"
t983ud , "axaxby"
